_test/common_functions/hashable_obj_tester.m

```diff
@@ -0,0 +1,56 @@
+function hashable_obj_tester(hobj,values,names)
+%hashable_obj_tester Generic function which validates correct hashable
+%object operations.
+%
+% Checks if all hashable properties invalidate hash, hash calculations work
+% and hash is stored-restored correctly
+%
+% Inputs:
+% hobj  -- instance of hashable object, better with proper values set
```

Comment:
unclear what is meant by better. What will happen if they aren't all set

Reply:
hopefully clarified